run_tests.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I wonder if we should add a `bin` directory, since my guess is over time we will have a number of scripts assembled like this. Though not an issue for now.